pac4j-cas/src/test/java/org/pac4j/cas/client/CasClientTests.java

```diff
@@ -127,7 +127,7 @@ public void testBackLogout() {
         final MockWebContext context = MockWebContext.create().addRequestParameter(CasConfiguration.LOGOUT_REQUEST_PARAMETER, LOGOUT_MESSAGE)
             .setRequestMethod("POST");
         TestsHelper.expectException(() -> casClient.getCredentials(context), HttpAction.class, "back logout request: no credential returned");
-        assertEquals(200, context.getResponseStatus());
+        assertEquals(204, context.getResponseStatus());
```

Comment:
@mmoayyed I'm wondering if returning a 204 (instead of 200) on CAS logout and proxy requests will cause issues with the CAS server.